app/jobs/algolia/remove_from_index_job.rb

```diff
@@ -0,0 +1,9 @@
+module Algolia
+  class RemoveFromIndexJob < ApplicationJob
+    queue_as :remove_from_algolia_index
```

Comment:
Perhaps rename the index to `search_remove_from_index`. We haven't been so good so far at following this convention but it'd be nice to have queues grouped in the list.

Basically I'm suggesting using the namespace name as a prefix and the name of the job without the 'job' part as name of the queue